azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -402,11 +403,10 @@ private void getToken(final String userToProxy) throws InterruptedException,
     }
   }
 
-  private void cancelHiveToken(final Token<? extends TokenIdentifier> t,
-      final String userToProxy) throws HadoopSecurityManagerException {
+  private void cancelHiveToken(final Token<? extends TokenIdentifier> t)
+      throws HadoopSecurityManagerException {
```

Comment:
"This one, for example, has been lurking around for a long time and I haven't touched it because I wanted to make a refactor PR that never happened." How can I help so that you can make it happen more often in the future? :) Any suggestion?